### ISSUE TITLE
API to get metaKGP org stats

### ISSUE THREAD

**Issue by Majestic9169** [Majestic9169/metakgp-website#5]
we need stats for 
- lines of code
- contributors
- projects (repos) 

of the metakgp org

**Comment by Waqibsk** [comment]
i have a solution for this issue:

1. When the app starts ,we will fetch the required data (from metakgp/repos github api)
2. Save the data in localstorage.
3. When navigating between pages , we will  retrieve the cached data in local storage instead of fetching it again from the API.

Looking forward for your suggestions...

**Comment by Ananya-m0140** [comment]
@Waqibsk Yes, you can go ahead. If you face some issue with the local storage part, you can also specify a time threshold after which data will be fetched again from API.

**Comment by Majestic9169** [comment]
That part is fine, my issue was more like I didn't really find an api to get the stats of a github organisation, just github users

Do let us know what you find and what your plan of action will be, once you have a plan you can create a draft pr
